requirements_c.txt

```diff
@@ -4,7 +4,12 @@
 #
 #########################################################
 nimfa==1.2.3
-numpy==1.10.4
-openslide-python==1.1.0
```

Comment:
There are still functions in core HistomicsTK that use openslide. Maybe leave it in there?